### ISSUE TITLE
Bootstrap MD Breakpoint 782px?

### ISSUE THREAD

**Issue by rikkilake** [WordPress/gutenberg#13775]
**Describe the bug**

source file: /wp-includes/css/dist/block-library/style.min.css?ver=5.0.3

On line 148: It says 782px instead of the more universally accepted 768px.  Until this gets fixed is there a safe way to remove/replace these media queries from the dist/ with a hook or similiar so template/child-templates can take charge?

`@media (min-width: 782px) {
    .wp-block-columns {
        flex-wrap:nowrap
    }
}`

**Expected behavior**
The media query should be updated to be universally compatible with such frameworks as Bootstrap, Bulma et al.

i.e.

`@media (min-width: 768px) {
    .wp-block-columns {
        flex-wrap:nowrap
    }
}`

**Desktop (please complete the following information):**
 - Windows 7
 - Chrome (latest)

**Additional context**
- Gutenberg 5.0.3

**Comment by youknowriad** [comment]
The breakpoints where chosen based on WordPress Core previous usage. Not sure if there's anything we can do here. I'm leaning towards just closing. 

Curious why benefit do we have in using the same breakpoints as Bootstrap ... or any other CSS framework? These styles can still be overriden using theme editor styles.

cc @jasmussen

**Comment by rikkilake** [comment]
"chosen based on WordPress previous usage" interesting if true, I always thought 768px was a widespread breakpoint.

Yes you are right I can easily over-ride it, but all breakpoints "expose" themselves in Chrome Dev Tools; like to keep things tidy.

![image](https://user-images.githubusercontent.com/18682923/52486795-02a53580-2bb4-11e9-9ffe-64f9ab3c02a9.png)

**Comment by jasmussen** [comment]
Hi @rikkilake, thanks for the suggestion. Riad is right, the breakpoints are based on existing core usage. 

I would like for us to do an audit of all breakpoints in WordPress and move towards more standardized ones. This is a big project, though, as it touches every single screen in WordPress, and over the years quite a few queries have built up. Here's a screenshot of all the queries in WordPress:

<img width="910" alt="screenshot 2019-02-11 at 09 23 44" src="https://user-images.githubusercontent.com/1204802/52551994-f8b34a80-2dde-11e9-81fa-14bec69d2e9b.png">

You can see the variables we extracted at the top, those are the most used ones. 

In other words, I don't think there's anything _currently_ actionable in this ticket. But long term it seems like it would be good to:

a) move all of WordPress to use the same reduced set of breakpoints
b) consider moving that reduced set, once consistent, towards more widely adopted breakpoints.

**Comment by youknowriad** [comment]
Thanks @jasmussen for the details. Let's close this issue for the moment.
